CDP4JsonFileDal/JsonFileDalUtils.cs

```diff
@@ -39,12 +39,12 @@ internal static class JsonFileDalUtils
         /// <summary>
         /// A remark to be included in the exchange header file.
         /// </summary>
-        private const string ExchangeHeaderRemark = "This is an ECSS-E-TM-10-25 exchange file";
+        public const string ExchangeHeaderRemark = "This is an ECSS-E-TM-10-25 exchange file";
 
         /// <summary>
-        /// The copyright text to be included in the exchange header file.
+        /// The default copyright text to be included in the exchange header file.
         /// </summary>
-        private const string ExchangeHeaderCopyright = "Copyright 2016 © RHEA.";
+        public const string DefaultHeaderCopyright = "Copyright 2016 © RHEA.";
```

Comment:
I would suggest naming both variables using the style `DefaultExchangeHeaderRemark` and `DefaultExchangeHeaderCopyright`.